tests/e2e/classic_ingress_test.go

```diff
@@ -40,22 +49,8 @@ var _ = Describe("Ingress updating", ci.NonHCPCluster, func() {
 				Cluster:          clusterID,
 			}
 			err = ingressService.Apply(&args)
-			if profile.GetClusterType().HCP {
```

Comment:
If this part is removed, we cannot detect the error message of HCP ingress updating anymore.

Reply:
Classic and HCP ingresses are now tested separately. Do we want to test incompatibility between classic and hcp specific resources ?

Reply:
For the resource, the checkpoint on HCP is not for the resource itself. But to check if errors happen, like HCP cluster reject the ingress updating if terraform provider can exit with backend error. There was a bug before that terraform provider cannot exit with some error happened, it stuck

Reply:
not sure I got the point.
Should I leave it here ?
I mean, yeah, HCP and Classic have different ingress fields for same API endpoint, but do we want to test cross-fields ?
Then should I add a check on HCP ingress resource testing to make sure that if classic cluster, then it fails ?

Reply:
@radtriste Yeah, let's leave it there. It is used to verify the happened error on HCP.

Reply:
Ok. With https://github.com/terraform-redhat/terraform-provider-rhcs/pull/632 I deactivated this case for HCP ingress, I guess I need to set it back also.

Reply:
restored